velox/common/memory/SharedArbitrator.h

```diff
@@ -282,12 +292,29 @@ class SharedArbitrator : public memory::MemoryArbitrator {
       uint64_t& maxGrowTarget,
       uint64_t& minGrowTarget);
 
-  // Invoked to get or refresh the memory stats of the candidate memory pools
-  // for arbitration. If 'freeCapacityOnly' is true, then we only get free
-  // capacity stats for each candidate memory pool.
-  void getCandidateStats(
-      ArbitrationOperation* op,
-      bool freeCapacityOnly = false);
+  // Invoked to get or refresh the candidate memory pools for arbitration. If
+  // 'freeCapacityOnly' is true, then we only get free capacity stats for each
+  // candidate memory pool.
+  void getCandidates(ArbitrationOperation* op, bool freeCapacityOnly = false);
```

Comment:
maybe call it updateCanditates

Reply:
It is to get candidates on the first call so it is better to keep is as getCandidates with comment.